### ISSUE TITLE
Run Llama.cpp in silent mode

### ISSUE THREAD

**Issue by MathiasSchindler** [ggerganov/llama.cpp#8507]
### Discussed in https://github.com/ggerganov/llama.cpp/discussions/777

<div type='discussions-op-text'>

<sup>Originally posted by **OsaCode** April  5, 2023</sup>
Is there an option to run the executable using a saved prompt without printing anything but the prediction by the selected model ? Some kind of silent mode in opposition to the --verbose.</div>

I believe this is a good first issue to introduce a --silent flag.

**Comment by AmgadHasan** [comment]
Running it with `--log-disable --no-display-prompt` flags does what you want
```sh
 $ ~/llama.cpp/b3405/bin/llama-cli -m ./models/Lite-Mistral-150M-v2-Instruct-Q8_0.gguf -p "<s>system
You are a helpful assistant</s>
<s>user
Write an article about Deep Learning. The article should be 1000 words at least</s> 
<s>assistant
" -n 1024 --log-disable --no-display-prompt

Deep learning has revolutionized the way we learn and use artificial intelligence. It has revolutionized many industries, including healthcare, finance, and manufacturing. In this article, we'll explore the fascinating process behind deep learning, covering its various applications, and discuss its potential as a powerful tool for enhancing human intelligence.
```

**Comment by nh2** [comment]
This is broken again. On version `8983`, `--log-disable --no-display-prompt` prints the full banner:

```
Loading model...  


▄▄ ▄▄
██ ██
██ ██  ▀▀█▄ ███▄███▄  ▀▀█▄    ▄████ ████▄ ████▄
██ ██ ▄█▀██ ██ ██ ██ ▄█▀██    ██    ██ ██ ██ ██
██ ██ ▀█▄██ ██ ██ ██ ▀█▄██ ██ ▀████ ████▀ ████▀
                                    ██    ██
                                    ▀▀    ▀▀

build      : b8983-80afa33
model      : gemma-4-E2B-it-Q4_0.gguf
modalities : text, vision, audio

available commands:
  /exit or Ctrl+C     stop or exit
  /regen              regenerate the last response
  /clear              clear the chat history
  /read <file>        add a text file
  /glob <pattern>     add text files using globbing pattern
  /image <file>       add an image file
  /audio <file>       add an audio file


> Test

```

**Comment by nh2** [comment]
PR with a fix: #22848

**Comment by sanmai** [comment]
What commit broke it?
